src/templates/index.html

```diff
@@ -4,7 +4,7 @@
         {% if session['role'] == "admin"%}
             <p>Welcome Sir!</p>
         {% elif session['role'] == "player"%}
-            <p>Form up Grunt!</p>
+            {% include 'user_menu.html' %}
```

Comment:
Maybe put the user menu in the layout so that it is automatically included on all pages.